changelog.d/3975.feature

```diff
@@ -1 +1 @@
-First user should autocreate autojoin rooms
+Servers with auto join rooms, should autocreate those rooms when first user registers
```

Comment:
s/auto join/auto-join/

s/should/will now/

s/first user/the first user/

possibly, s/autocreate/automatically create/